crates/core/component/dex/src/component/position_manager/counter.rs

```diff
@@ -11,6 +11,7 @@ use anyhow::Result;
 pub(crate) trait PositionCounterRead: StateRead {
     /// Returns the number of position for a [`TradingPair`].
     /// If there were no counter initialized for a given pair, this default to zero.
+    #[cfg(test)]
```

Comment:
```suggestion
```

we need this method for the eviction logic, is it leaking in the warnings?

Reply:
yeah, in a galileo build i was seeing warnings that this isn't used. this likely just needs a `#[cfg(feature = "?")]` feature gate

Reply:
got it, i think this might have been fixed when we did the component api clean up for the eviction pr

Reply:
oh nice! thank you very much. [three steps ahead](https://www.youtube.com/watch?v=KB09paeV_aE), as always :slightly_smiling_face: :musical_note: